includes/Core/Authentication/Authentication.php

```diff
@@ -443,6 +450,23 @@ public function get_disconnect_url() {
 		);
 	}
 
+	/**
+	 * Gets the proxy permission URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string Proxy permission URL.
+	 */
+	private function get_proxy_permissions_url() {
+		return add_query_arg(
+			array(
+				'action' => Google_Proxy::ACTION_PERMISSIONS,
+				'nonce'  => wp_create_nonce( Google_Proxy::ACTION_PERMISSIONS ),
+			),
+			admin_url( 'index.php' )
+		);
+	}
```

Comment:
See my previous review, this method should also be moved to the end of the class where the similar proxy setup methods are located.